monkey/infection_monkey/monkey.py

```diff
@@ -68,8 +79,7 @@ def __init__(self, args):
     def initialize(self):
         logger.info("Monkey is initializing...")
 
-        if not self._singleton.try_lock():
-            raise Exception("Another instance of the monkey is already running")
+        self._check_for_running_monkey()
```

Comment:
I think this belongs in the `start` or `setup`. Exiting if another monkey is running is a behavior/logic and initialization is not for that